install/install.sh

```diff
@@ -240,9 +253,7 @@ confirm "Install Starship ${GREEN}latest${NO_COLOR} to ${BOLD}${GREEN}${BIN_DIR}
 
 info "Installing Starship, please wait…"
```

Comment:
It may make more sense if we could escalate privileges before saying "Installing, please wait..."

Maybe the flow should be:
```sh
confirm "Install Starship ${GREEN}latest${NO_COLOR} to ${BOLD}${GREEN}${BIN_DIR}${NO_COLOR}?"
check_permissions
info "Installing Starship, please wait…"
install
complete "Starship installed"
```

What are your thoughts?

Reply:
Yeah I agree, will move the *info* to install function =)